### PR TITLE
Clusters post list layout

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
New Features:
* A `research_clusters` layout for the `ucf-post-list` shortcode has been added.
* Fallback header image settings have been added to the customizer for the research clusters.

Small Changes:
* The name of the research cluster field tabs have been shortened to help them fit on the screen better.
* A short description field has been added to allow for the small blurb above the "Learn more about..." button on the post list layout.

**Motivation and Context**
This allows the clusters to be listed without having to provide manual markup for each one on the listing page.

**How Has This Been Tested?**

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: includes/config.php [hunk 1]
```diff
@@ -41,6 +41,40 @@ function research_define_customizer_fields( $wp_customize ) {
...
+	);
+
+	$wp_customize->add_control(
+		new WP_Customize_Image_Control(
+			$wp_customize,
+			'cluster_fallback_bg',
+			array(
+				'label'    => __( 'Upload a fallback header image for clusters' ),
+				'section'  => RESEARCH_THEME_CUSTOMIZER_PREFIX . 'clusters',
+				'height'   => 1600,
```
Comment (cjg89) [inline includes/config.php:56]: Height and width values appear to be inverted.  Also, the height for header images inherited from the UCF WP Theme is 500, not 550--not sure if this is an intentional change.

File: includes/post-list-functions.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				<picture>
+					<source media="(min-width: 575px)" srcset="<?php echo $bg_image_sm; ?>">
+					<img src="<?php echo $bg_image_xs; ?>" alt="" class="co-jumbotron-bg media-background object-fit-cover">
+				</picture>
+				<?php endif; ?>
+				<div class="container">
+					<div class="row p-lg-5">
+						<div class="col-12 col-md-8<?php echo $push ? ' offset-md-4' : ''?>">
+							<div class="bg-inverse-t-3 p-5">
+								<h2 class="display-5 font-condensed text-uppercase mt-0 mb-4" id="<?php echo $post->post_name; ?>-heading" aria-label="<?php echo $post->post_title; ?>"><?php echo $post->post_title; ?></h2>
```
Comment (cjg89) [inline includes/post-list-functions.php:41]: `display-5` class doesn't exist

File: includes/config.php (hunk 1 above)
Comment (jmbarne3) [reply includes/config.php]: Updated the constraints.. I'll have to update the header images I've source so far for this.

File: includes/post-list-functions.php (hunk 2 above)
Comment (jmbarne3) [reply includes/post-list-functions.php]: Removed the class for now. We can use another sizing class if we need to.

**Pull request merged**